src/main/features/claim/guards/newFeaturesConsentGuard.ts

```diff
@@ -0,0 +1,19 @@
+import * as express from 'express'
+import { ClaimStoreClient } from 'claims/claimStoreClient'
+import { ForbiddenError } from 'errors'
+
+const claimStoreClient = new ClaimStoreClient()
+
+export class NewFeaturesConsentGuard {
+
+  static requestHandler (req: express.Request, res: express.Response, next: express.NextFunction) {
+    claimStoreClient.retrieveUserRoles(res.locals.user)
```

Comment:
Please use GuardFactory so that we have single way of creating guards and no extra steps are needed in a11y tests to mock them.